zulipterminal/cli/run.py

```diff
@@ -117,8 +117,17 @@ def parse_zuliprc(zuliprc_str: str) -> Dict[str, Any]:
         except EOFError:
             pass
 
+    fail_msg = "\nFailed to parse zuliprc file at " + zuliprc_path + "\n"
     zuliprc = configparser.ConfigParser()
-    zuliprc.read(zuliprc_path)
+
+    try:
+        res = zuliprc.read(zuliprc_path)
+        if len(res) == 0:
+            print(in_color('red', fail_msg))
+            sys.exit(1)
+    except configparser.MissingSectionHeaderError:
+        print(in_color('red', fail_msg))
+        sys.exit(1)
```

Comment:
Could be — but I think we'd need to throw an error more general than `configparser.MissingSectionHeaderError` to handle it outside the function. What do you think?

Reply:
Regarding handling it differently, I think for now it would be fine as it is.

However, I'm rereading the code, and it's unclear to me under what conditions each of these fail - and this is a general error message, which won't help the user either. Could you confirm under what condition each occurs? Maybe output a different message, if the errors are different?

Reply:
These two cases are, in fact, slightly different.

The first one corresponds to a problem with file access: e.g. a directory was passed as a file path, or there's no read access to the file.
The second one is for parsing failures — e.g. when an arbitrary file was passed as a configuration file.

Reply:
Sure, I can add a comment, but would you like these two cases to be handled differently, or can it stay as it is?

Reply:
If you have identified the specifics, I think it would be useful for users to know the details of the errors.